docs/application/native/guides/multimedia/pose-detection.md

```diff
@@ -0,0 +1,343 @@
+    error_code = mv_engine_config_set_string_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_INPUT_DATA_TYPE,
+    MV_INFERENCE_DATA_FLOAT32);
+
+    error_code = mv_engine_config_set_string_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_MODEL_USER_FILE_PATH,
+    MODEL_MAPPING_FILE);
+
+    error_code = mv_engine_config_set_double_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_MODEL_MEAN_VALUE,
+    0.0);
+
+    error_code = mv_engine_config_set_double_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_MODEL_STD_VALUE,
+    1.0);
+
+    error_code = mv_engine_config_set_int_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_BACKEND_TYPE,
+    MV_INFERENCE_BACKEND_TFLITE);
+
+    error_code = mv_engine_config_set_int_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_BACKEND_TYPE,
+    MV_INFERENCE_TARGET_CPU);
+
+    error_code = mv_engine_config_set_int_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_INPUT_TENSOR_WIDTH,
+    192);
+
+    error_code = mv_engine_config_set_int_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_INPUT_TENSOR_HEIGHT,
+    192);
+
+    error_code = mv_engine_config_set_int_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_INPUT_TENSOR_CHANNELS,
+    3);
+
+    error_code = mv_engine_config_set_string_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_INPUT_NODE_NAME,
+    inputNodeName);
+
+    error_code = mv_engine_config_set_array_string_attribute(imagedata.g_engine_config,
+    MV_INFERENCE_OUTPUT_NODE_NAMES,
+    outputNodeName, 1);
+    ```
+
+    For more information on the configuration attributes such as `MV_INFERENCE_MODEL_WEIGHT_FILE_PATH`, see Media Vision Inference API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) applications).
+
+5. Use `mv_inference_configure()` to configure `g_inference` inference handle with `g_engine_config`:
+
+    ```c
+    error_code = mv_inference_configure(imagedata.g_inference, imagedata.g_engine_config);
+    if (error_code != MEDIA_VISION_ERROR_NONE)
+      dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+    ```
+
+6. Use `mv_inference_prepare()` to prepare inference:
+
+    ```c
+    error_code = mv_inference_prepare(imagedata.g_inference);
+    if (error_code != MEDIA_VISION_ERROR_NONE)
+      dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+      ```
+
+7. To detect pose, create `g_pose` media vision pose handle:
+
+    ```c
+    error_code = mv_pose_create(&imagedata.g_pose);
+    if (error_code != MEDIA_VISION_ERROR_NONE)
+      dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+      ```
+
+8. Suppose that MoCap BVH file `mocap.bvh` and its mapping file `mocap_mapping.txt` are used as the same ones described in [Background](#background) and they are in `<OwnDataPath>`. Then, set them to `g_pose` handle to compare and detect the pose:
```

Comment:
@tyoungroy 
I need your opinion on the following: 
How about slight modification? Please verify for technical correctness too. 
```suggestion 
8. Suppose that MoCap BVH file `mocap.bvh` and its mapping file `mocap_mapping.txt` are used as per the description in the [Background](#background)section, and the files are stored in `<OwnDataPath>`. Then, set the files to `g_pose` handle for comparing and detecting the pose:

Just a question: 
them refers to ?

Reply:
How about changing as follows:
```suggestion
8. Suppose that MoCap BVH file `mocap.bvh` and its mapping file `mocap_mapping.txt`, which are described in the [Background](#background) section, are applied and the files are stored in `<OwnDataPath>`. Then, set the files to `g_pose` handle for comparing and detecting the pose:
```

'them' means the files 'mocap.bvh' and ' mocap_mapping.txt'. They are set to 'g_pose' handle and the they are used to compare the pose with the results from mv_inference_pose_landmark_detect()